server/routes/therapist.js

```diff
@@ -4,58 +4,81 @@ const therapistRoutes = express()
 
 // db models
 const db = require('../../db/')
-const patientModel  = db.model('patient') 
-const therapistModel  = db.model('therapist') 
+const therapistModel  = db.model('therapist')
+const patientModel  = db.model('patient')
+const exerciseModel  = db.model('exercise') 
 
 // -=-=-= CREATE =-=-=-
 
 // create therapist
 therapistRoutes.post('/', (req, res, next) => {
-	therapistRoutes.create({
-	     first_name: req.body.first_name,
-	     last_name: req.body.last_name,
-	     practice_name: req.body.practice_name,
-	     license_id: req.body.license_id,
-	     img_URL: req.body.image_URL,
-	     email: req.body.email,
-	     gender: req.body.gender,
-	     password: req.body.password
-   	})
-	.then(therapist => res.send(therapist))
-	.catch(next);
+  therapistModel.create({
```

Comment:
Consistency with Patient route wrt snake case vs. camel case? Should we stick with snake for back, camel for front, or just snake across the board?